packages/core/stories/spinner/spinner.stories.tsx

```diff
@@ -41,6 +50,67 @@ Large.args = {
   size: "large",
 };
 
+export const Loading: Story = () => {
+  const [exampleIsLoading, setExampleIsLoading] = useState(true);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setExampleIsLoading(false);
+    }, 5000);
+  });
+
+  return (
+    <>
+      <div style={{ textAlign: "center" }}>
+        <p>Please wait for action to complete.</p>
+        {exampleIsLoading ? (
+          <Spinner
+            style={{ margin: "auto" }}
+            aria-label="Panel is loading"
+            size="large"
+          />
+        ) : (
+          <p>Action complete.</p>
```

Comment:
Is it worth adding a reload button. Right now if you don't scroll down within 5 seconds, you will not see the spinner.